include/wlr/interfaces/wlr_output.h

```diff
@@ -23,7 +23,7 @@ struct wlr_output_impl {
 	bool (*attach_render)(struct wlr_output *output, int *buffer_age);
 	bool (*test)(struct wlr_output *output);
 	bool (*commit)(struct wlr_output *output);
-	void (*rollback)(struct wlr_output *output);
+	void (*rollback_render)(struct wlr_output *output);
```

Comment:
A bit out of scope for this change, but do you think you could go through and add doc comments to each of these fields?

Reply:
Good idea. Done in https://github.com/swaywm/wlroots/pull/2293